mmv1/products/workstations/api.yaml

```diff
@@ -0,0 +1,189 @@
+        input: true
+        url_param_only: true
+        description: |
+          The location where the workstation cluster should reside.
+    properties:
+      - !ruby/object:Api::Type::String
+        name: "name"
+        output: true
+        description: |
+          The name of the cluster resource.
+      - !ruby/object:Api::Type::String
+        name: "uid"
+        output: true
+        description: |
+          The system-generated UID of the resource.
+      - !ruby/object:Api::Type::KeyValuePairs
+        name: "labels"
+        description: "Client-specified labels that are applied to the resource and that are also propagated to the underlying Compute Engine resources."
+      - !ruby/object:Api::Type::String
+        name: "network"
+        required: true
+        input: true
+        description: |
+          The relative resource name of the VPC network on which the instance can be accessed. 
+          It is specified in the following form: "projects/{projectNumber}/global/networks/{network_id}".
+      - !ruby/object:Api::Type::String
+        name: "subnetwork"
+        required: true
+        input: true
+        description: |
+          Name of the Compute Engine subnetwork in which instances associated with this cluster will be created. 
+          Must be part of the subnetwork specified for this cluster.
+      - !ruby/object:Api::Type::String
+        name: "displayName"
+        description: |
+          Human-readable name for this resource.
+      - !ruby/object:Api::Type::Boolean
+        name: "degraded"
+        description: |
+          Whether this resource is in degraded mode, in which case it may require user action to restore full functionality. 
+          Details can be found in the conditions field.
+        output: true
+      - !ruby/object:Api::Type::Boolean
+        name: "reconciling"
+        description: |
+          Whether this resource is in degraded mode, in which case it may require user action to restore full functionality. 
+          Details can be found in the conditions field.
+        output: true
+      - !ruby/object:Api::Type::KeyValuePairs
+        name: "annotations"
+        description: "Client-specified annotations. This is distinct from labels."
+      - !ruby/object:Api::Type::String
+        name: "etag"
+        description: |
+          Checksum computed by the server. 
+          May be sent on update and delete requests to ensure that the client has an up-to-date value before proceeding.
+      - !ruby/object:Api::Type::Time
+        name: "createTime"
+        description: |
+          Time the Instance was created in UTC.
+        output: true
+      - !ruby/object:Api::Type::Time
+        name: "updateTime"
+        description: |
+          Time the Instance was updated in UTC.
+        output: true
+      - !ruby/object:Api::Type::Time
+        name: "deleteTime"
+        description: |
+          Time the Instance was deleted in UTC.
+        output: true
```

Comment:
These two fields are irrelevant and can be removed.

Reply:
Are you referring to updateTime and deleteTime? If so, I have removed these :+1:

Reply:
correct !